CMakeLists.txt

```diff
@@ -9,14 +9,19 @@
 
 cmake_minimum_required( VERSION 3.12 )
 
+#find_package( ecbuild 3.4 REQUIRED HINTS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild )
```

Comment:
Is this needed? If not, please delete.

Reply:
I kept this as a reminder when we move to `ecbuild 3.4` to uncomment this and remove the next line.
I am happy to delete.
ecbuild 3.4 will simplify/impact a lot of places.